pkg/console/operator/sync_v400.go

```diff
@@ -61,6 +65,7 @@ func (co *consoleOperator) sync_v400(updatedOperatorConfig *operatorv1.Console,
 	if rtErr != nil {
 		return rtErr
 	}
+	co.CheckRouteHealth(updatedOperatorConfig, rt)
```

Comment:
I'm inclined to move this to the route controller here:

https://github.com/openshift/console-operator/blob/master/pkg/console/controllers/route/controller.go#L109

After the sync.